sdks/python/apache_beam/transforms/util.py

```diff
@@ -864,3 +867,225 @@ def add_window_info(element, timestamp=DoFn.TimestampParam,
 
     def expand(self, pcoll):
       return pcoll | ParDo(self.add_window_info)
+
+
+@ptransform_fn
+def matches_all_object(pcoll, regex, group=None):
```

Comment:
Do we expect these to be public interfaces?

If not, can we prefix them with `_` and move them under Regex.

Reply:
Nope, it should be private. Let me fix this.